src/app/components/section-display/section-display.component.html

```diff
@@ -1,7 +1,12 @@
 <div class="container">
     <div class="section-display">APPLICATION</div>
     <div class="log-out">
-       <button class="log-out-btn"><img src="../../assets/power.png" alt=""></button>
-       <p> Logout</p>
+       <a 
+        routerLink=""
```

Comment:
empty routerLink

Reply:
I will correct this...on clicking the logout, it takes the user to the login page back.